### PR TITLE
Support `int` index in `MultiEmbeddingTensor`

### PR DESCRIPTION
**Pull request by akihironitta**
Part of #88.

```python
import torch
from torch_frame.data import MultiEmbeddingTensor
tensor_list = [
    torch.tensor([[0, 1, 2], [6, 7, 8]]),  # col1
    torch.tensor([[3, 4], [9, 10]]),       # col2
    torch.tensor([[5], [11]]),             # col3
]
met = MultiEmbeddingTensor.from_tensor_list(tensor_list)
print(met)
# MultiEmbeddingTensor(num_rows=2, num_cols=3, device='cpu')
print(met[0])
# MultiEmbeddingTensor(num_rows=1, num_cols=3, device='cpu')
print(met[0].values)
# tensor([[0, 1, 2, 3, 4, 5]])
print(met[0].offset)
# tensor([0, 3, 5, 6])
```

### CODE REVIEW

File: torch_frame/data/multi_embedding_tensor.py [hunk 1]
```diff
@@ -196,3 +206,32 @@ def cat(
...
+        dim: int,
+        is_slice_end: bool = False,
+    ) -> Union[int, Tensor]:
+        """Helper function to map negative indices to positive indices and
+        raise :obj:`IndexError` when necessary.
+
+        Args:
+            index: Union[int, Tensor]: Input :obj:`index` with potentially
+                negative elements.
+            is_slice_end (bool): Whether a given index (int) is slice or not.
```
Comment (zechengz) [inline torch_frame/data/multi_embedding_tensor.py:222]: Though not very sure, should it be `is slice end or not`?
```suggestion
            is_slice_end (bool): Whether a given index (int) is slice end or not.
```

**Review by zechengz — APPROVED** [review]

File: torch_frame/data/multi_embedding_tensor.py [hunk 2]
```diff
@@ -196,3 +206,32 @@ def cat(
             # which is inconsistent with when dim=0
             offset = torch.LongTensor(offset_list)
             return MultiEmbeddingTensor(num_rows, num_cols, values, offset)
+
+    def _normalize_index(
+        self,
+        index: Union[int, Tensor],
+        dim: int,
+        is_slice_end: bool = False,
```
Comment (weihua916) [inline torch_frame/data/multi_embedding_tensor.py:214]: We already have `_normalize_index` in `MultiNestedTensor`. https://github.com/pyg-team/pytorch-frame/blob/master/torch_frame/data/multi_nested_tensor.py#L373-L409

What's the difference?

Comment (akihironitta) [reply torch_frame/data/multi_embedding_tensor.py:214]: It's exactly the same. I'm moving it to the parent class

Comment (weihua916) [reply torch_frame/data/multi_embedding_tensor.py:214]: Can we instead move `_normalize` to `_MultiTensor`, so that it can be used for both classes?

Comment (weihua916) [reply torch_frame/data/multi_embedding_tensor.py:214]: Super, thanks.

File: torch_frame/data/multi_embedding_tensor.py (hunk 1 above)
Comment (akihironitta) [reply torch_frame/data/multi_embedding_tensor.py]: Done!

File: torch_frame/data/multi_embedding_tensor.py (hunk 2 above)
Comment (akihironitta) [reply torch_frame/data/multi_embedding_tensor.py]: Done! PTAL :)

File: torch_frame/data/multi_embedding_tensor.py (hunk 1 above)
Comment (zechengz) [reply torch_frame/data/multi_embedding_tensor.py]: Thanks!

**Review by weihua916 — APPROVED** [review]
Thanks!

**Pull request merged**